<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Parameters - MODAS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "Parameters";
    var mkdocs_page_input_path = "parameters.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MODAS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="..">Home</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../installation/">Installation</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../quickstart/">QuickStart</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../tutorial/">Tutorial</a>
                    </li>
                </ul>
                <ul class="current">
                    <li class="toctree-l1 current"><a class="reference internal current" href="./">Parameters</a>
    <ul class="current">
    <li class="toctree-l2"><a class="reference internal" href="#genotype-dimensionality-reduction-module-genoidx">Genotype Dimensionality Reduction Module (genoidx)</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#phenotype-preprocessing-and-transformation-module-phenorm">Phenotype preprocessing and transformation module (phenorm)</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#omics-phenotype-pre-screening-module-prescreen">Omics Phenotype Pre-screening Module (prescreen)</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#regional-gwas-identify-qtl-module-regiongwas">Regional gwas identify QTL module  (regiongwas)</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#mendelian-randomization-analysis-modulemr">Mendelian randomization analysis module(mr)</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#whole-genome-wide-association-analysis-and-visualization-modulevisual">Whole genome-wide association analysis and visualization module(visual)</a>
    </li>
    </ul>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../visual_example/mainPage.html">Visual_example</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MODAS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Parameters</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h1 id="parameters">Parameters</h1>
<p>This chapter introduces the parameters of each module in detail. </p>
<h2 id="genotype-dimensionality-reduction-module-genoidx">Genotype Dimensionality Reduction Module (genoidx)</h2>
<p><code>-g</code>: Genotype file used for genome-wide genotype dimensionality reduction analysis, support plink bed format and hapmap formatï¼Œthe plink bed format file only needs to provide the file prefix, the hapmap format needs to be used together with the parameter <code>-convert</code><br/>
<code>-genome_cluster</code>: Perform genome-wide genotype dimensionality reduction analysis on genotype file<br/> 
<code>-w</code>: Window size for genome-wide genotype dimensionality reduction analysis(bp), default 1000000<br/>
<code>-s</code>: Step size for genome-wide genotype dimensionality reduction analysis(bp), default 500000<br/>
<code>-convert</code>: Convert hapmap genotype format to plink bed format genotype<br/>
<code>-clump</code>: Clumping analysis for genotype file,  used to keep a subset of SNPs that are nearly uncorrelated with each other, thereby reducing the number of total SNPs to speed up genome-wide genotype dimensionality reduction analysis <br/>
<code>-p</code>: Number of threads for genome cluster analysis,max threads is total number of genome chromosomes<br/>
<code>-o</code>: The prefix of the output file, default MODAS_genoidx_output</p>
<h2 id="phenotype-preprocessing-and-transformation-module-phenorm">Phenotype preprocessing and transformation module (phenorm)</h2>
<p><code>-phe</code>: Phenotype file for phenotype preprocessing and transformation, the file format is csv format, the first column and the first row are the names of inbred lines and phenotypes respectively<br/>
<code>-g</code>: Genotype file in plink bed format for principal component analysis, used with the parameter <code>-pca</code> to correct differences in phenotype caused by population structure<br/>
<code>-pca</code>: Correct the differences in phenotype caused by population structure through PCA<br/>
<code>-r</code>: Phenotype missing ratio filter cutoff<br/>
<code>-v</code>: Phenotype value filter cutoff<br/>
<code>-log2</code>: log2 transformation of phenotype<br/>
<code>-log10</code>: log10 transformation of phenotype<br/>
<code>-ln</code>: ln transformation of phenotype<br/>
<code>-qqnorm</code>: Normal quantile transformation of phenotype<br/>
<code>-norm</code>: Boxcox transformation of phenotype<br/>
<code>-o</code>: The prefix of the output file, default MODAS_phenorm_output<br/></p>
<h2 id="omics-phenotype-pre-screening-module-prescreen">Omics Phenotype Pre-screening Module (prescreen)</h2>
<p><code>-g</code>: Genotype file in plink bed format, used to calculate the kinship matrix for GWAS analysis<br/>
<code>-genome_cluster</code>: Pseudo-genotype file for phenotype pre-screening, generated by subcommand <code>genoidx</code><br/>
<code>-phe</code>: Phenotype file for phenotype pre-screening analysis, the file format is csv format, the first column and the first row are the names of inbred lines and phenotypes respectively <br/>
<code>-lm_suggest_pvalue</code>: Linear model suggest pvalue for pseudo-SNP filter used for association analysis in mixed linear model, default 1/ pseudo-genotype file pseudo-snp number<br/>
<code>-lmm_suggest_pvalue</code>: LMM suggest pvalue for phenotype prescreen, used to obtain candidate phenotypes with significant association analysis signals and significant association regions of phenotypes, default 1e-5<br/>
<code>-p</code>: Number of threads for analysis in prescreen sub command<br/>
<code>-o</code>: The prefix of the output file, default MODAS_prescreen_output</p>
<h2 id="regional-gwas-identify-qtl-module-regiongwas">Regional gwas identify QTL module  (regiongwas)</h2>
<p><code>-g</code>: Genotype file in plink bed format, used to calculate the kinship matrix for regional gwas analysis and extract snp in significant association regions of phenotypes <br/>
<code>-phe</code>: Candidate phenotype file generated by subcommand <code>prescreen</code> <br/>
<code>-phe_sig_qtl</code>: Significant association regions of candidate phenotypes file generated by subcommand <code>prescreen</code><br/>
<code>-p1</code>: Significance threshold for index SNPs, used to determine the phenotype with QTL and index snps of phenotype<br/>
<code>-p2</code>: Secondary significance threshold for clumped SNPs, Used to obtain the secondary significant SNP linked to index snp to determine the reliability of the QTL. MODAS outputs the QTL containing 10 secondary significant SNPs as the phenotypic QTL result<br/>
<code>-cluster</code>: Cluster phenotype by QTL position<br/>
<code>-p</code>: Number of threads for regional gwas analysis, default 1<br/>
<code>-o</code>: The prefix of the output file, default MODAS_regiongwas_out<br/></p>
<h2 id="mendelian-randomization-analysis-modulemr">Mendelian randomization analysis module(mr)</h2>
<p><code>-g</code>: Genotype file in plink bed format<br/>
<code>-exposure</code>: Exposure phenotype file, such as mTrait phenotype, the file format is csv format, the first column and the first row are the names of inbred lines and phenotypes respectively<br/>
<code>-outcome</code>: Outcome phenotype file, such as pTrait phenotype,  the file format is csv format, the first column and the first row are the names of inbred lines and phenotypes respectively<br/>
<code>-qtl</code>: Exposure phenotype QTL file, generated by subcommand <code>regiongwas</code><br/>
<code>-lm</code>: Perform Mendelian Randomization through linear model<br/>
<code>-mlm</code>: Perform Mendelian Randomization through mixed linear model<br/>
<code>-p</code>: Number of threads for Mendelian Randomization analysis<br/>
<code>-pvalue</code>: The pvalue cutoff of Mendelian randomization analysis result output,default 1 <br/>
<code>-net</code>: Mendelian Randomization network analysis, used to identify functional modules<br/>
<code>-o</code>: The prefix of the output file, default MODAS_mr_out</p>
<h2 id="whole-genome-wide-association-analysis-and-visualization-modulevisual">Whole genome-wide association analysis and visualization module(visual)</h2>
<p><code>-g</code>: Genotype file in plink bed format for whole genome-wide association analysis<br/>
<code>-phe</code>: Phenotype file for GWAS analysis and visualization<br/>
<code>-qtl</code>: Phenotype QTL file generated by subcommand <code>regiongwas</code><br/>
<code>-visual</code>: Generate web-based GWAS visualization results<br/>
<code>-anno</code>: Gene annotation file for QTL annotation, used to display QTL information in visualized results<br/>
<code>-p</code>: Number of threads genome-wide association analysis and visualization<br/>
<code>-o</code>: The prefix of the output file,default MODAS_visual_out</p>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
      
        <a href="../tutorial/" class="btn btn-neutral" title="Tutorial"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
        <span><a href="../tutorial/" style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
    </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
