<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>QuickStart - MODAS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "QuickStart";
    var mkdocs_page_input_path = "quickstart.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MODAS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="..">Home</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../installation/">Installation</a>
                    </li>
                </ul>
                <ul class="current">
                    <li class="toctree-l1 current"><a class="reference internal current" href="./">QuickStart</a>
    <ul class="current">
    <li class="toctree-l2"><a class="reference internal" href="#input-format">Input format</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#generate-pseudo-genotype-files">Generate pseudo-genotype files</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#prescreen-candidate-genomic-regions-for-omics-data">Prescreen candidate genomic regions for omics data</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#perform-regional-association-analysis-to-identify-qtls">Perform regional association analysis to identify QTLs</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#perform-mendelian-randomization-analysis">Perform Mendelian randomization analysis</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#whole-genome-wide-association-analysis-and-visualization">Whole genome-wide association analysis and visualization</a>
    </li>
    </ul>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../tutorial/">Tutorial</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../parameters/">Parameters</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../visual_example/mainPage.html">Visual_example</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MODAS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>QuickStart</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h1 id="quick-start">Quick Start</h1>
<p>The basic use of MODAS generally includes three steps: first, generate pseudo-genotype files; second, use the pseudo-genotype files to screen candidate genomic blocks; third, identify QTLs by SNP-based regional association analysis. In addition, Mendelian randomization (MR) analysis and GWAS visualization can also be performed for specified traits. In this part, only basic commands are listed and briefly described for a quick start, please refer to the tutorial part for a more detailed description of the implementation principle and parameters of MODAS.</p>
<h2 id="input-format">Input format</h2>
<h3 id="genotype-format">Genotype format</h3>
<p>MODAS takes genotype files in plink-bed format as input, it also supports the conversion of hapmap format to plink-bed format. For usage, refer to the subcommand <code>genoidx</code>.</p>
<h3 id="phenotype-format">Phenotype format</h3>
<p>MODAS takes omics phenotype files in csv format as input, the first column and the first row are the names of inbred lines and phenotypes respectively.</p>
<h2 id="generate-pseudo-genotype-files">Generate pseudo-genotype files</h2>
<p>The pseudo-genotype files can be generated from the <code>-genome_cluster</code> parameter in the subcommand <code>genoidx</code>. Since the genotype files are relatively large, <code>-p</code> is generally added to specify multi-threading to accelerate the generation of pseudo-genotype files.</p>
<pre><code class='nohighlight'>MODAS.py genoidx -g ./chr_HAMP -genome_cluster -p 10 -o chr_HAMP
</code></pre>
<h2 id="prescreen-candidate-genomic-regions-for-omics-data">Prescreen candidate genomic regions for omics data</h2>
<p>Prescreening of candidate genomic regions for omics data/molecular traits (mTraits) is generated by the subcommand <code>prescreen</code>. In this step, the pseudo-genotype files are first used to perform association analysis on the mTraits through linear model (LM), general linear model(GLM) or  mixed linear model (MLM). Then, the significantly associated genomic regions (SAGRs) for each mTrait are used as the candidate genomic regions.</p>
<pre><code class='nohighlight'>MODAS.py prescreen -g ./chr_HAMP -genome_cluster ./chr_HAMP.genome_cluster.csv -phe ./E3_log2.normalized_phe.csv -gwas_model MLM -p 20 -o E3_log2
</code></pre>
<h2 id="perform-regional-association-analysis-to-identify-qtls">Perform regional association analysis to identify QTLs</h2>
<p>The QTL identification step is generated by the subcommand <code>regiongwas</code>. In this step, the SNPs from the candidate genomic regions are extracted to perform regional association analysis through LM, GLM or MLM, and the QTLs for each mTrait are identified according to the results of SNP-based regional association analysis.</p>
<pre><code class='nohighlight'>MODAS.py regiongwas -g ./chr_HAMP -phe ./E3_log2.sig_omics_phe.csv -phe_sig_qtl ./E3_log2.phe_sig_qtl.csv -gwas_model MLM -p 20 -o E3_log2
</code></pre>
<h2 id="perform-mendelian-randomization-analysis">Perform Mendelian randomization analysis</h2>
<p>Mendelian randomization (MR) analysis is generated by the subcommand <code>mr</code>. In this step, the peak SNP of a QTL for each trait is used to infer the causal relationship between trait pairs. MR analysis in MODAS can be performed using either linear model (LM) or mixed linear model (MLM), with parameters <code>-lm</code> and <code>-mlm</code>, respectively.</p>
<pre><code class='nohighlight'># lm model
MODAS.py mr -g chr_HAMP -exposure AMP_kernel_transcriptome_v4_FPKM_correct.sig_eqtl.qqnorm.csv -outcome blup_traits_final.new.csv -qtl AMP_kernel_transcriptome_qtl_res.csv -lm  -o AMP_kernel_transcriptome_MR_lm

# mlm model
MODAS.py mr -g chr_HAMP -exposure AMP_kernel_transcriptome_v4_FPKM.sig_eqtl.qqnorm.csv -outcome blup_traits_final.new.csv -qtl AMP_kernel_transcriptome_qtl_res.csv -mlm  -o AMP_kernel_transcriptome_MR_mlm
</code></pre>
<h2 id="whole-genome-wide-association-analysis-and-visualization">Whole genome-wide association analysis and visualization</h2>
<p>GWAS visualization is generated by the subcommand visual. In this step, MODAS uses specified QTL and trait files as inputs, to perform GWAS and generate whole-genome level Manhattan plots, and displays the results through a HTML based web page.</p>
<pre><code class='nohighlight'># visualization
MODAS.py visual -g  chr_HAMP -phe E3_log2.normalized_phe.csv -qtl E3_log2.local_gwas_qtl_res.csv -gwas_model gemma_MLM -p 6 -visual -anno maize_genefunc.txt -o E3_log2_visual
</code></pre>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../tutorial/" class="btn btn-neutral float-right" title="Tutorial">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href="../installation/" class="btn btn-neutral" title="Installation"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
        <span><a href="../installation/" style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
        <span style="margin-left: 15px"><a href="../tutorial/" style="color: #fcfcfc">Next &raquo;</a></span>
      
    </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
